test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/NuGet.PackageManagement.UI.Test.csproj

```diff
@@ -7,6 +7,8 @@
     <NETCoreWPFProject>true</NETCoreWPFProject>
     <TestProject>true</TestProject>
     <Description>Unit and integration tests for NuGet.PackageManagement.UI.</Description>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
```

Comment:
Isn't this true by default?

Reply:
I followed the instructions to put this here. I'd rather keep it explicit.

Reply:
Which instructions? 
When it comes to the csproj the defaults are there for a reason and they are not changed often because they wouldn't want to break customers.

Reply:
https://aka.ms/vssdktestfx

Reply:
You don't need it for SDK based projects. I think the instructions can be improved there. 

See relevant info: https://github.com/dotnet/sdk/blob/b865dde7ff6a8ad0cc0f5b748f24d47265570a8c/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.BeforeCommon.targets#L81-L84

Reply:
I err on the side of being explicit, rather than relying on defaults. I don't see the cost in this case to leaving it. Maybe others can chime in?

Reply:
The whole idea of SDK based projects is to remove the fluff from the project file. 
That's the improvement of these project files over the old csproj project files. 

The automatic binding redirect generation is just one of many, many defaults we rely on in the new projects compared to the old projects.